parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/bridge_hub_config.rs

```diff
@@ -0,0 +1,213 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Bridge definitions.
+
+use crate::{
+	BridgeParachainPolkadotInstance, Runtime, WithBridgeHubPolkadotMessagesInstance, XcmRouter,
+};
+use bp_messages::LaneId;
+use bridge_runtime_common::{
+	messages,
+	messages::{
+		source::FromBridgedChainMessagesDeliveryProof, target::FromBridgedChainMessagesProof,
+		MessageBridge, ThisChainWithMessages, UnderlyingChainProvider,
+	},
+	messages_xcm_extension::{XcmBlobHauler, XcmBlobHaulerAdapter},
+	refund_relayer_extension::{
+		ActualFeeRefund, RefundBridgedParachainMessages, RefundableMessagesLane,
+		RefundableParachain,
+	},
+};
+use frame_support::{parameter_types, RuntimeDebug};
+use xcm::{latest::prelude::*, prelude::NetworkId};
+use xcm_builder::{BridgeBlobDispatcher, HaulBlobExporter};
+
+parameter_types! {
+	pub const MaxUnrewardedRelayerEntriesAtInboundLane: bp_messages::MessageNonce =
+		bp_bridge_hub_kusama::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
+		bp_bridge_hub_kusama::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+	pub const BridgeHubPolkadotChainId: bp_runtime::ChainId = bp_runtime::BRIDGE_HUB_POLKADOT_CHAIN_ID;
+	pub PolkadotGlobalConsensusNetwork: NetworkId = NetworkId::Polkadot;
+	pub PriorityBoostPerMessage: u64 = 921_900_294;
```

Comment:
Please use something like:
```rust
const FEE_BOOST_PER_MESSAGE: Balance = <some-value>;
bridge_runtime_common::priority_calculator::ensure_priority_boost_is_sane::<
	Runtime,
	WithBridgeHubPolkadotMessagesInstance,
	PriorityBoostPerMessage,
>(FEE_BOOST_PER_MESSAGE);
```
to compute and test this value. The `FEE_BOOST_PER_MESSAGE` should be fee that the transaction fee that relayer must add to the message delivery transaction with `N` messages to make its priority equal to delivery transaction with `N+1` messages. It should be large enough to prevent single-message delivery transactions when there are many queued messages at the bridged chain. My guess - e.g. `5 DOT` and `1 KSM` accordingly - this would need an approval from other reviewers and then properly audited as well.

Reply:
Fixed